app/assets/scss/application.scss

```diff
@@ -95,15 +95,17 @@ $path: "/admin/static/images/";
   margin-bottom: 40px;
 }
 
-.summary-item-field-first {
-  width: 15%;
-}
+.fixed-column-width-table {
+  .summary-item-field-first {
+    width: 15%;
+  }
 
-.summary-item-field {
-  width: 15%;
-  padding-left: 10px;
-}
+  .summary-item-field {
+    width: 15%;
+    padding-left: 10px;
```

Comment:
I think we probably want to keep the padding-left for all summary table fields, not just the `fixed-column-width-table`.  I reckon the only thing inside the `fixed-column-width-table` block should be the 15% width rules.